### ISSUE TITLE
predefined drivers configuration not fully working

### ISSUE THREAD

**Issue by andrew-g-za** [dbeaver/dbeaver#8654]
#### System information: 
- Operating system: Windows Server 2019 standard
- DBeaver version: 7.0.3.202004191823
- Additional extensions: none

#### Connection specification:
- Database name and version: Postgres
- Driver name: org.postgresql.Driver (postgresql-42.2.12.jar)
- Do you use tunnels or proxies (SSH, SOCKS, etc): no

#### Describe the problem you're observing:
I am trying to set up a Dbeaver installation for all users logging into the server.
The server is on the internal network and cannot access the internet, so i have to pre-configure
the database driver to connect to our postgres database.

The pre-configured driver.xml file does not fully work - when starting up, dbeaver still inserts maven settings into the driver config. 

Note that the server supports multiple simultaneous users via RDP - any of whom should be able to start dbeaver and set up a connection to postgres without having to download or configure the driver.

#### Steps to reproduce, if exist:
Install dbeaver using the default settings.

Following the section 'Provide predefined drivers configuration' in the [guide ](https://dbeaver.com/docs/wiki/Admin-Manage-Drivers/), create a drivers.xml file in `C:\Program Files\DBeaver\shared_drivers` :

```
<?xml version="1.0" encoding="UTF-8"?>
<drivers>
	<provider id="wmi"/>
	<provider id="athena"/>
	<provider id="db2"/>
	<provider id="db2_zos"/>
	<provider id="db2_i"/>
	<provider id="elasticsearch"/>
	<provider id="exasol"/>
	<provider id="jaybird"/>
	<provider id="generic"/>
	<provider id="bigquery"/>
	<provider id="google_spanner"/>
	<provider id="greenplum"/>
	<provider id="h2gis"/>
	<provider id="hana"/>
	<provider id="kognitio"/>
	<provider id="sqlserver"/>
	<provider id="mssql"/>
	<provider id="mysql"/>
	<provider id="ocient"/>
	<provider id="open-distro-elasticsearch"/>
	<provider id="oracle"/>
	<provider id="postgresql">
		<driver id="postgres-jdbc" categories="sql" custom="false" embedded="false" name="PostgreSQL" class="org.postgresql.Driver" url="jdbc:postgresql://{host}[:{port}]/[{database}]" port="5432" description="PostgreSQL standard driver">
			<library type="jar" path="C:\Program Files\DBeaver\shared_drivers\postgresql-42.2.12.jar" custom="true"/>
		</driver>
	</provider>
	<provider id="snowflake"/>
	<provider id="vertica"/>
</drivers>

```
Edit the `dbeaver.ini` file to point it to the driver config:

```
-startup
plugins/org.eclipse.equinox.launcher_1.5.700.v20200207-2156.jar
--launcher.library
plugins/org.eclipse.equinox.launcher.win32.win32.x86_64_1.1.1100.v20190907-0426
-vmargs
-XX:+IgnoreUnrecognizedVMOptions
--add-modules=ALL-SYSTEM
-Xms64m
-Xmx1024m
-Ddbeaver.drivers.configuration-file=C:\Program Files\DBeaver\shared_drivers\drivers.xml
```

Start Dbeaver, then navigate to Database > Driver Manager > PostgreSQL > Edit

Under the libraries tab, the maven co-ordinates for the drivers appear, and are ahead of the driver file (which does display as the 3rd item in the list).

However, when trying to create a connection using the driver, dbeaver still attempts to connect to maven. Since the server is not connected to the internet, this results in an error.

The point being however - i have explicitly provided a driver.xml configuration file, so the maven co-ordinates should NOT be added?

**Comment by serge-rider** [comment]
Thanks for the bug report. Indeed, driver library config must be fully replaced when it is specified thru `dbeaver.drivers.configuration-file` parameter. Now configs are merged so Maven artifact remains in the driver.

**Comment by andrew-g-za** [comment]
No worries.
Happy to help out if this is a suitable beginner task - just point me in the right direction and i'll see if i can make the change.

**Comment by uslss** [comment]
verified

**Comment by LonwoLonwo** [comment]
fixed in 7.1.0
